pytealutils/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .blob import *
```

Comment:
If the idea is for this repo to be a collection of subpackages with a low barrier to entry, I'd recommend not importing all subpackages here, since there's likely to be name collisions or unnecessary exports, meaning more work will be required to review PRs that add new subpackages.

Instead this file could be blank, and in order to use a subpackage, you could do either of the following:

```python
from pytealutils import packageA, packageB

packageA.PackageADefinition(...)
packageB.PackageBDefinition(...)
```

OR

```python
from pytealutils.packageA import PackageADefinition
from pytealutils.packageB import PackageBDefinition

PackageADefinition(...)
PackageBDefinition(...)
```